src/libraries/System.Numerics.Vectors/System.Numerics.Vectors.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I guess these changes to the sln file are not intentional?

Reply:
Visual Studio made those changes automatically (first commit in the PR).

Reply:
I think these changes occur when a newer version of VS opens the sln. AFAIK, reverting these changes work, so that's what I've always done.

Reply:
It's kind of annoying having to revert the changes every time I have to submit a new commit. @safern would I break something if I keep the sln changes?

Reply:
I'll keep this unless it causes any issues.